hist/histv7/test/add.cxx

```diff
@@ -46,10 +53,6 @@ TEST(HistAddTest, AddHist) {
    hFrom.Fill({0.1111}, .42f);
    hTo.Fill({0.1111}, .17f);
    ROOT::Experimental::Add(hTo, hFrom);
+   EXPECT_EQ(2, hTo.GetEntries());
    EXPECT_FLOAT_EQ(0.59f, hTo.GetBinContent({0.1111}));
 }
-
-// Test addition of a hist range
-TEST(HistAddTest, AddView) {
-  EXPECT_EQ(1, 1);
-}
```

Comment:
Let's be honest here, that's not a real test. I could accept a RODO, FIXME, or Jira ticket as a replacement.